docs/tutorials/installation.md

```diff
@@ -0,0 +1,302 @@
+# How to install JAX
+
```

Comment:
You previously mentioned the idea of putting the most common commands up top. How about adding something like this here?

> Most jax installation will look something like this:
> ```
> # CPU: Linux/OSX/Windows
> $ pip install "jax[cpu]"
>
> # GPU: Linux with CUDA 12
> pip install "jax[cuda11_pip]" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
> ```
> For other platforms and architectures, see the sections below.